sklearnex/preview/linear_model/ridge.py

```diff
@@ -0,0 +1,422 @@
+                alpha=1.0,
+                fit_intercept=True,
+                copy_X=True,
+                max_iter=None,
+                tol=1e-4,
+                solver="auto",
+                positive=False,
+                random_state=None,
+            ):
+                super().__init__(
+                    alpha=alpha,
+                    fit_intercept=fit_intercept,
+                    copy_X=copy_X,
+                    max_iter=max_iter,
+                    tol=tol,
+                    solver=solver,
+                    positive=positive,
+                    random_state=random_state,
+                )
+
+        else:
+
+            def __init__(
+                self,
+                alpha=1.0,
+                fit_intercept=True,
+                normalize="deprecated" if sklearn_check_version("1.0") else False,
+                copy_X=True,
+                max_iter=None,
+                tol=1e-4,
+                solver="auto",
+                positive=False,
+                random_state=None,
+            ):
+                super().__init__(
+                    alpha=alpha,
+                    fit_intercept=fit_intercept,
+                    normalize=normalize,
+                    copy_X=copy_X,
+                    max_iter=max_iter,
+                    solver=solver,
+                    tol=tol,
+                    positive=positive,
+                    random_state=random_state,
+                )
+
+        def fit(self, X, y, sample_weight=None):
+            # It is necessary to properly update coefs for predict if we
+            # fallback to sklearn in dispatch
+            if hasattr(self, "_onedal_estimator"):
+                del self._onedal_estimator
+
+            dispatch(
+                self,
+                "fit",
+                {
+                    "onedal": self.__class__._onedal_fit,
+                    "sklearn": sklearn_RidgeRegression.fit,
+                },
+                X,
+                y,
+                sample_weight,
+            )
+            return self
+
+        @wrap_output_data
+        def predict(self, X):
+            check_is_fitted(self)
+
+            return dispatch(
+                self,
+                "predict",
+                {
+                    "onedal": self.__class__._onedal_predict,
+                    "sklearn": sklearn_RidgeRegression.predict,
+                },
+                X,
+            )
+
+        @wrap_output_data
+        def score(self, X, y, sample_weight=None):
+            check_is_fitted(self)
+
+            return dispatch(
+                self,
+                "score",
+                {
+                    "onedal": self.__class__._onedal_score,
+                    "sklearn": sklearn_RidgeRegression.score,
+                },
+                X,
+                y,
+                sample_weight=sample_weight,
+            )
+
+        def _test_type_and_finiteness(self, X_in):
```

Comment:
```suggestion
        def _check_type_and_finiteness(self, X_in):
```

Reply:
using prefix "_test" to conform to sklearnex.linear_model.logistic_regression.py/linear.py style using the same function definition

Reply:
It is strange, that they using "_test" prefix. But I think we can go with you proposal then. Thank you!